config_test.go

```diff
@@ -32,7 +32,8 @@ var testConfig = `{
   "PollIntervalSeconds": 60,
   "ServerURL": "mender.io",
   "DeviceID": "1234-ABCD",
-  "ServerCertificate": "/data/server.crt"
+  "ServerCertificate": "/data/server.crt",
+  "UpdateLogPath": "/data/logs/update.log"
```

Comment:
I think this should be `/var/lib/mender` and `/var/lib/mender/log` (for the certificate as well). Yes, I know, it doesn't really matter in a test, but I think we should stick to what we're gonna use. Less confusing when reading it later.